app/src/test/java/tttTDD/Java/GameTest.java

```diff
@@ -137,11 +164,11 @@ public void GameShouldListIfTheGameHasConcludedAndWinner() {
         int computerInput2 = 2;
         int humanInput3 = 6;
 
-        game.performMove(humanInput1);
-        game.performMove(computerInput1);
-        game.performMove(humanInput2);
-        game.performMove(computerInput2);
-        game.performMove(humanInput3);
+        game.performTurn(humanInput1);
```

Comment:
I completely understand where you're coming from with this. The reason being was at the time this test was written, it needed to be tested without a game loop. Although it may not seem apparent, the test name gives a description as to what's happening with the code. One thing about this though, the computer input isn't necessary as the board provides the available moves. It's there more for readability purposes 🤣